common/crypto/ledger/ledger_secp256k1.go

```diff
@@ -0,0 +1,98 @@
+package ledger
+
+import (
+	"github.com/binance-chain/go-sdk/common/crypto"
+	"github.com/binance-chain/go-sdk/common/crypto/secp256k1"
+	"github.com/binance-chain/go-sdk/types"
+	"github.com/btcsuite/btcd/btcec"
+	ledgergo "github.com/zondax/ledger-cosmos-go"
+)
+
+var (
+	// discoverLedger defines a function to be invoked at runtime for discovering
+	// a connected Ledger device.
+	DiscoverLedger discoverLedgerFn
+)
+
+type (
+
+	// discoverLedgerFn defines a Ledger discovery function that returns a
+	// connected device or an error upon failure. Its allows a method to avoid CGO
+	// dependencies when Ledger support is potentially not enabled.
+	discoverLedgerFn func() (LedgerSecp256k1, error)
+
+	// DerivationPath represents a Ledger derivation path.
+	DerivationPath []uint32
+
+	// LedgerSecp256k1 reflects an interface a Ledger API must implement for
+	// the SECP256K1 scheme.
+	LedgerSecp256k1 interface {
+		GetPublicKeySECP256K1([]uint32) ([]byte, error)
```

Comment:
`[]uint32` -> `DerivationPath`

Reply:
I can't do this. Otherwize, the implemenation will not be compatiable with this interface.

Reply:
Ok then my bad